docs/modules/ROOT/pages/erc721-uri-storage.adoc

```diff
@@ -40,7 +40,7 @@ impl Erc721MetadataExample {
     }
 
     pub fn burn(&mut self, token_id: U256) -> Result<(), Vec<u8>> {
-        Ok(self.erc721.burn(token_id)?)
+        Ok(self.erc721._burn(token_id)?)
```

Comment:
It works this way in the example -> https://github.com/OpenZeppelin/rust-contracts-stylus/blob/main/examples/erc721-metadata/src/lib.rs#L35

Reply:
Right, because of [the IErc721Burnable import](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/416/files#diff-a9b0af634cd736ef4664f7a7954e8c680b555025006fc8d2dc27a8de9e09ab6cR19)